scripts/mostro.sh

```
@@ -0,0 +1,129 @@
+  local index="${!#}"
+  private_key_from_mnemonic_path "$mnemonic" "$MOSTRO_DERIVATION_PATH/$index"
+}
+
+mostro_create_random_orders() {
+  info create random sell orders for alice
+  for i in 0 1 2; do
+    mostro_create_random_sell_order "$(alice_identity_private_key)"
+  done
+
+  info create random sell orders for bob
+  for i in 0 1 2; do
```

Shellcheck (0.10.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)